database.py

```diff
@@ -129,9 +131,9 @@ def add_user(self, student_id:str, username:str, admin_status:int):
             # user is an admin
             elif admin_status == 1:
                 cursor.execute('''
-                INSERT INTO all_users (student_id, username, admin_status, start_time, working_status, total_minutes)
-                VALUES (?, ?, ?, NULL, 0, 0)
-                ''', (student_id, username, admin_status))
+                INSERT INTO all_users (student_id, username, admin_status, start_time, working_status, total_minutes, graduation_year)
+                VALUES (?, ?, ?, NULL, 0, 0, ?)
```

Comment:
You should hardcode a value for grad year for admins. You can leave grad year as 0000 for admins because they're admins.